src/metabase/driver/bigquery.clj

```diff
@@ -351,7 +351,7 @@
         results (if mbql?
                   (post-process-mbql dataset-id table-name results)
                   (update results :columns (partial map keyword)))]
-    (assoc results :annotate? true)))
+    (assoc results :annotate? mbql?)))
```

Comment:
"Annotate" is the post-processing stage of the query processor that adds extra info about columns and sorts the results. It's only supposed to be done for MBQL queries, not native ones